apps/ledger-live-desktop/tests/specs/speculos/sendSpeculos.spec.ts

```diff
@@ -0,0 +1,75 @@
+import test from "../../fixtures/common";
+import { expect } from "@playwright/test";
+import { Layout } from "../../models/Layout";
+import { AccountsPage } from "../../models/AccountsPage";
+import { AccountPage } from "../../models/AccountPage";
+import { SendModal } from "../../models/SendModal";
+import { SpeculosModal } from "../../models/SpeculosModal";
+import { Modal } from "../../models/Modal";
+import { Currency } from "../../enum/Currency";
+
+test.use({ userdata: "speculos" });
+
+const currencies: Currency[] = [Currency.tETH];
+
+const addresses: { [key in Currency["uiName"]]: string } = {
+  [Currency.BTC.uiName]: "bc1q7ezsfc44adw2gyzqjmwhuh2e83uk8u5hrw590r",
+  [Currency.tBTC.uiName]: "tb1q8kkh3hkwaq6frqrfdkhpmxzzhe5dtclzwlu4y9",
+  [Currency.ETH.uiName]: "0x43047a5023D55a8658Fcb1c1Cea468311AdAA3Ad",
+  [Currency.tETH.uiName]: "0x43047a5023D55a8658Fcb1c1Cea468311AdAA3Ad",
+  [Currency.SOL.uiName]: "TODO",
```

Comment:
I feel like you could create an enum just like you did [here](https://github.com/LedgerHQ/ledger-live/pull/6680/files#diff-c8300da2d385b75f5a0bb39ceebdcc40e22af2bd87f23ebd2cdc2082fbac763aR1) (for reusability across different tests suites's scenarios)
also worth noting that those addresses correspond to an account index (I imagine first account for each currency on your seed)